components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/MarketplaceAssistantApiServiceImpl.java

```diff
@@ -0,0 +1,132 @@
+package org.wso2.carbon.apimgt.rest.api.store.v1.impl;
```

Comment:
Add the license header

Reply:
Fixed with f9052eea1b85d6e35e35bfb0c5a8ba69304cba05